### PR TITLE
fixed visual and logical errors described in #423

### PR DESCRIPTION
**Pull request by edewit**
fixing: #423

### CODE REVIEW

File: src/clients/ClientSettings.tsx [hunk 1]
```diff
@@ -141,13 +234,18 @@ export const ClientSettings = ({ save }: ClientSettingsProps) => {
...
+              }
             />
           </FormGroup>
           <ActionGroup className="keycloak__form_actions">
             <Button variant="primary" onClick={save}>
               {t("common:save")}
             </Button>
-            <Button variant="link">{t("common:cancel")}</Button>
+            <Button variant="link" onClick={reload}>
+              {t("common:reload")}
```
Comment (mfrances17) [inline src/clients/ClientSettings.tsx:247]: this was changed to 'revert' in a recent bug fix

**Review by mfrances17 — CHANGES_REQUESTED** [review]
Some quick changes:

- 'reload' is now 'revert' in common messages, as per recent bug fix
- he had asked for more spacing between the two columns of auth flow checkboxes, i think you missed it:
auth-flow-spacing

- the left margins for section heads are inconsistent between tabs - check out the difference between the settings tab and the advanced tab:
settings-left-margin

advanced-left-margin

**Review by jenny-s51 — CHANGES_REQUESTED** [review]
Looking good @edewit ! 

A couple small things:

1) The login theme dropdown has a "blank" option that is clickable ... instead we probably want something like the client type dropdown where the default state has the "Select an option" text:
Screen Shot 2021-03-16 at 2 51 30 PM
2) In the "Advanced" tab the scroll form needs left padding. 

This can be fixed by updating form-panel.css as shown below:

```
.kc-form-panel__panel {
  padding-top: var(--pf-global--spacer--lg);
  padding-bottom: var(--pf-global--spacer--2xl);
  padding-left: var(--pf-global--spacer--lg); /* add this line */
  max-width: 768px;
}
```

**Comment by edewit** [comment]
The left margins are fixed in the PR for the UXD review of the Advanced settings page

**Review by mfrances17 — APPROVED** [review]

**Review by jenny-s51 — APPROVED** [review]

**Pull request merged**